File: c-sharp/TwoSum.cs
int[] result = Solution.TwoSum([2, 7, 11, 15], 9);
Console.WriteLine(string.Join(", ", result));

public class Solution
{
    public static int[] TwoSum(int[] nums, int target)
    {
        List<int> summedArray = [];

        for (int i = 0; i < nums.Length; i++)
        {
            for (int j = i + 1; j < nums.Length; j++)
            {
                if (nums[i] + nums[j] == target && i != j && summedArray.Count < 2)
                {
                    summedArray.Add(i);
                    summedArray.Add(j);
                }
            }
        }

        return [.. summedArray];
    }
}

File: python/two_sum.py
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int):
        summed_arr = []
        for k, v in enumerate(nums):
            for j, w in enumerate(nums):
                if v + w == target and k != j and len(summed_arr) < 2:
                    summed_arr.extend([k, j])

        return summed_arr


s = Solution()
print(s.twoSum([2, 7, 11, 15], 9))
